lib/test_prof/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module TestProf
-  VERSION = "1.0.7"
+  VERSION = "1.0.8"
```

Comment:
Please, do not update the version in the PR. It should be done during release

Reply:
Oh, of course, I removed the commit that added this change.